pep-0682.rst

```diff
@@ -125,10 +130,11 @@ An optional, literal ``z`` is added to the
 
     [[fill]align][sign][z][#][0][width][grouping_option][.precision][type]
 
-where ``z`` is allowed for numerical types other than integer.  Support for
-``z`` is provided by the ``.__format__()`` method of each numeric type,
-allowing the specifier to be used in f-strings, built-in ``format()``, and
-``str.format()``.  The %-formatting style will not support the new option.
+where ``z`` is allowed for floating-point presentation types (``f``, etc.).
```

Comment:
> floating-point presentation types

I think the previous wording is clearer here -- what is a "floating-point presentation type"? The example of `f` etc also doesn't help, as f-strings are a mechanism by which to format things to strings, not themselves a type.

Is there a reason we shouldn't say "numerical types other than integer"?

Reply:
"presentation type" refers to the `type` field of the format specification, and is used extensively in the format specification documentation to avoid confusion with the type of the format input value.  `f` is an example presentation type.  Would "(`f`, `g`, etc.) be more clear?

Reply:
> Would "(f, g, etc.)" be more clear?

I've made that change for now

Reply:
"floating-point presentation types as defined by the format specification"? If the list of floating point presentation types is small enough, it might be better to just list them all exhaustivley -- i.e. `f`, `g`, `blah` as Python 3.11.

I didn't pick up that this was a reference to the format spec usage from the docs -- when I see `type` I read that as a object type by default. It also might be worth adding a sentence on the fact that input types get coerced depending on the presentation type, or just redirect to the docs here -- the distinction isn't obvious right away to those who haven't spent a lot of time with the formatting innards.

A

Reply:
I'm wondering if it's enough as is.  This PEP extends the Format Specification Mini-Language, and there is already a link to those docs.  The phrase "floating point presentation types" is used twice there, e.g. in defining the recently-added `'_'` option.

Even if the person evaluating the PEP isn't familiar with the mini-language documentation, "presentation type" seems a good hint that it's regarding the output type, and to further clarify I've given a few examples (`f`, `g`).  There are 9 floating point presentation types-- I'd rather not list them exhaustively.

Reply:
I've changed the wording to:

> (``f``, ``g``, etc.,  as defined by the format specification documentation)

Reply:
nine is too many, yes. I think your latest change is sufficient to point to the right direction -- thank you.

A